src/fold/composites/concat.py

```diff
@@ -101,7 +100,14 @@ def clone(self, clone_child_transformations: Callable) -> Concat:
 class Pipeline(Composite):
     """
     An optional wrapper that is equivalent to using a single array for the transformations.
-    It executes the transformations in the order they are provided.
+    It executes the transformations sequentially, in the order they are provided.
+
+    Parameters
+    ----------
+
+    pipelines : Pipelines
+        A list of pipelines to be applied to the data, independently of each other.
```

Comment:
I think it should take a single pipeline? but will fix it later :)
copilot gets confused easily.

Reply:
no this was right:)